src-data-processing/target_data_maker.R

```diff
@@ -0,0 +1,39 @@
+#' Target_data_maker
+#'
+#' A script to create data files with data available on a given date.
+
+require(tidyverse)
+require(arrow)
+# reading in the data; Here we are using the 100k sample to test instead of the full metadata.
+# If you want to use the full data
+# change the file path and use the commended #full_data_file <- read_delim_arrow(file_path, delim = "\t") line,
+# because of the different file types.
+file_path <- "./input-data/100Ksample.tsv.xz"
+full_data_file <- read_tsv(file_path)
+#full_data_file <- read_delim_arrow(file_path, delim = "\t")
+
+# removing any data before 2022, non-humans, and not in the 52 regions we care about
+full_data_file <- filter(full_data_file, date_submitted >= as.Date("2022-01-01"),
```

Comment:
I would suggest merging this and the next set of lines that are doing data processing all into one single tidyverse-style piped together set of commands. Not an obligatory change, but more of a style thing.